cmd/docker/main.go

```diff
@@ -164,6 +164,7 @@ func startServer() *apiserver.Server {
 	}
 
 	// FIXME: assignment copies lock value to tlsConfig: crypto/tls.Config contains sync.Once contains sync.Mutex
+	// #nosec
```

Comment:
@caglar10ur I'm not familiar with the implications, and that line seems to be coming from https://github.com/vmware/vic/pull/1994/files#diff-f346ba150f298a88e29c9f07d68d8c7aR179. The `tls.Config` has two fields that the FIXME comment mentions:
```go
	serverInitOnce sync.Once // guards calling (*Config).serverInit

	// mutex protects sessionTicketKeys and originalConfig.
	mutex sync.RWMutex
```

However, we don't seem to be copying those fields - `tlsconfig.ServerDefault()` doesn't set those fields.